Docs: remove IRC remnants (#709)
ansible/ansible-builder: CONTRIBUTING.md

# Ansible Builder Contributing Guidelines

If you are looking for community support, please visit
the [Community guide](https://ansible.readthedocs.io/projects/builder/en/latest/community/)

> for information how to reach out to us.

Reviewer: ```suggestion
for information on how to get in touch.
```
